LEGO1/legonavcontroller.cpp

```diff
@@ -155,7 +155,7 @@ float LegoNavController::CalculateNewTargetSpeed(int p_pos, int p_center, float
   float result;
   int diff = p_pos - p_center;
 
-  if (diff > this->m_mouseDeadzone)
+  if (diff < this->m_mouseDeadzone)
```

Comment:
This makes no sense, logically speaking. The function has already been matching 100% before, I suggest double checking this.

Reply:
The diff was showing jge vs jle so I thought I'd check it out anyways

Reply:
Was the diff also showing the registers flipped? Sometimes the compiler will flip the operation around but also the order of the registers, so logically it will be the same conditional

Reply:
Considering I cant find the original diff ill revert this